docs/self-hosting/configuration/envars.mdx

```diff
@@ -328,6 +328,25 @@ SMTP_FROM_NAME=Infisical
   </Info>
 </Accordion>
 
+<Accordion title="SMTP2Go"> 
+1. Create an account and configure [SMTP2Go](https://www.smtp2go.com/) to send emails.
+2. Turn on SMTP authentication  
+```
+SMTP server: mail.smtp2go.com
```

Comment:
Please follow the other email providers structure so that it is clear what environment variables should be added